components/esp_lvgl_port/src/lvgl8/esp_lvgl_port.c

```diff
@@ -198,10 +198,10 @@ static void lvgl_port_task(void *arg)
             task_delay_ms = lv_timer_handler();
             lvgl_port_unlock();
         }
-        if ((task_delay_ms > lvgl_port_ctx.task_max_sleep_ms) || (1 == task_delay_ms)) {
```

Comment:
` || (1 == task_delay_ms)` this part of code was recommendation from LVGL for right handling sleep. Why did you remove it?

Reply:
ok, we can keep it as it is, when there is better performance :-)